test/Markdown-test.ts

```diff
@@ -13,14 +13,10 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 */
-import * as linkifyjs from 'linkifyjs';
-import Markdown from "../src/Markdown";
-import matrixLinkify from '../src/linkify-matrix';
 
-beforeAll(() => {
-    // We need to call linkifier plugins before running those tests
-    matrixLinkify(linkifyjs);
-});
+// Properly import
+import '../src/linkify-matrix';
+import Markdown from "../src/Markdown";
```

Comment:
Markdown already imports linkify-matrix, seems redundant to import it here too?

Reply:
IMHO doesn't hurt, especially that it shows a direct dependency relation this way. Maybe I should make the comment smarter though.

Reply:
Just got rid of that.